### PR TITLE
Feature/navigation

### PR DESCRIPTION
**Pull request by jabbarn**
created navigation to the following screens:
- Home
- Friends
- Library
    - Profile and back
    - Settings and back

implemented basic theming. default for now is dark.
At last added dependencies for navigation. expo navigation bar and material ui icons.

### CODE REVIEW

File: frontend/Screens/Home.tsx [hunk 1]
```diff
@@ -57,10 +55,9 @@ const Home = ({navigation}) => {
     // }, [])
 
     return <View>
-        <Text>you're logged in</Text>
+        <Text>you're logged in!</Text>
         <Button onPress={() => logOut()} title={'logOut'}/>
         <Button onPress={() => navigation.navigate('Landingpage')} title={'go back to landingPage'}/>
```
Comment (Imadbk7) [inline frontend/Screens/Home.tsx]: this button does not redirect me to the landing page even though I am logged in

File: frontend/Screens/Home.tsx [hunk 2]
```diff
@@ -57,10 +55,9 @@ const Home = ({navigation}) => {
     // }, [])
 
     return <View>
-        <Text>you're logged in</Text>
+        <Text>you're logged in!</Text>
         <Button onPress={() => logOut()} title={'logOut'}/>
```
Comment (Imadbk7) [inline frontend/Screens/Home.tsx]: this button is almost invisible on an iphone device. It is also no longer clickable due to the notch that an iPhone has

Comment (jabbarn) [reply frontend/Screens/Home.tsx]: send screenshot

File: frontend/Screens/Home.tsx (hunk 1 above)
Comment (jabbarn) [reply frontend/Screens/Home.tsx]: removed it since landing page was more of a test, kept the actual file in screen for future use.

**Comment by jabbarn** [comment]
@Imadbk7 can you recheck and make an 'approving review'.

File: frontend/HomeTabs.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
+import React from 'react';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import Friends from './Screens/Friends';
+import Home from './Screens/Home';
+import LibraryStackNavigation from './LibraryStackNavigation';
+
+const Tab = createMaterialBottomTabNavigator();
+
+//@ts-ignore
```
Comment (Edens-Angel) [inline frontend/HomeTabs.tsx:10]: tsignore, why?

File: frontend/HomeTabs.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+const HomeTabs = ({navigation}): JSX.Element => {
+    return (
+        <Tab.Navigator
+            id="bottom-tab-navigator"
+            initialRouteName="home"
+            barStyle={{ backgroundColor: 'black' }}
+        >
+            <Tab.Screen
```
Comment (Edens-Angel) [inline frontend/HomeTabs.tsx:18]: In deze file kan je beter een array maken met alle tabs en die dan 1 keer loopen. Dan als je later iets wilt toevoegen hoef je dat alleen maar te doen in de array en geen html de gaan adden met dinge

File: frontend/Screens/Home.tsx [hunk 5]
```diff
@@ -1,41 +1,11 @@
...
-import {Alert, Button, Text, View} from "react-native";
-import {CLIENT_ID} from '@env'
+import {Alert, Button, View} from "react-native";
+import { Text } from "react-native-paper";
 import {useCookies} from "react-cookie";
-import axios from "axios";
 import {LoginContext} from "../Context";
-import {useNavigation} from "@react-navigation/native";
 
 // @ts-ignore
```
Comment (Edens-Angel) [inline frontend/Screens/Home.tsx:7]: Ts ignore?

File: frontend/Screens/LandingPage.tsx [hunk 6]
```diff
@@ -11,15 +11,13 @@ export const LandingPage = ({navigation}) => {
     // @ts-ignore
```
Comment (Edens-Angel) [inline frontend/Screens/LandingPage.tsx:11]: Tsignore?

File: frontend/Screens/Login.tsx [hunk 7]
```diff
@@ -1,8 +1,6 @@
-import {StatusBar} from "expo-status-bar";
 import React, {useContext, useEffect, useState} from "react";
-import {Image, KeyboardAvoidingView, StyleSheet, Text, View} from "react-native";
-import axios from "axios";
-import {Button} from "react-native-paper";
+import {Image, KeyboardAvoidingView, StyleSheet} from "react-native";
+import {Button, Text} from "react-native-paper";
 import {Cookies, useCookies} from 'react-cookie';
 
 // @ts-ignore
```
Comment (Edens-Angel) [inline frontend/Screens/Login.tsx:6]: tsignore?

File: frontend/Screens/Login.tsx [hunk 8]
```diff
@@ -17,8 +15,7 @@ const discovery = {
         "https://accounts.spotify.com/api/token",
 };
 
-// @ts-ignore
-const LoginScreen = ({navigation}) => {
+const LoginScreen = ({navigation}): JSX.Element => {
     const [cookies, setCookie, removeCookie] = useCookies(['loginCookie']);
     const [token, setToken] = useState("");
```
Comment (Edens-Angel) [inline frontend/Screens/Login.tsx:20]: In react best practice is NULL datatype te gebruiken in plaats van lege string.

**Review by Edens-Angel — APPROVED** [review]

**Pull request merged**